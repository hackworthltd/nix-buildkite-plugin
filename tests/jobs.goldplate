{
    "command": "nix-buildkite",
    "arguments": ["jobs.nix"],
    "asserts": [
        {"exit_code": 0},
        {"stdout": "${GOLDPLATE_NAME}.stdout"}
    ]
}

