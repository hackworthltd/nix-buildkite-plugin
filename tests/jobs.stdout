{"agents":{},"steps":[{"command":"nix-store -r /nix/store/kb5m9n3npjrwb8a8my9k9pqz7qwq1l0r-hello-2.12.1.drv","depends_on":[],"key":"kb5m9n3npjrwb8a8my9k9pqz7qwq1l0r-hello-2_12_1","label":":darwin: aarch64-darwin:hello-2.12.1"},{"command":"nix-store -r /nix/store/zna38ypg3k9ahc0hdk1ykk5gxdxwrh30-synthetic-deps.drv","depends_on":["kb5m9n3npjrwb8a8my9k9pqz7qwq1l0r-hello-2_12_1"],"key":"zna38ypg3k9ahc0hdk1ykk5gxdxwrh30-synthetic-deps","label":":darwin: aarch64-darwin:synthetic-deps"},{"command":"nix-store -r /nix/store/94mjmvv6cv9z6m8hil8r0s3b33glq23j-hello-2.12.1.drv","depends_on":[],"key":"94mjmvv6cv9z6m8hil8r0s3b33glq23j-hello-2_12_1","label":":linux: aarch64-linux:hello-2.12.1"},{"command":"nix-store -r /nix/store/a691rhfnwbqx2q3jpj0hkbf8c47pjw4j-synthetic-deps.drv","depends_on":["94mjmvv6cv9z6m8hil8r0s3b33glq23j-hello-2_12_1"],"key":"a691rhfnwbqx2q3jpj0hkbf8c47pjw4j-synthetic-deps","label":":linux: aarch64-linux:synthetic-deps"},{"command":"nix-store -r /nix/store/4xngvxjdd7fa3bqxzd6nwv6fh1hb4ry3-test-docker-image.tar.gz.drv","depends_on":["a691rhfnwbqx2q3jpj0hkbf8c47pjw4j-synthetic-deps","94mjmvv6cv9z6m8hil8r0s3b33glq23j-hello-2_12_1"],"key":"4xngvxjdd7fa3bqxzd6nwv6fh1hb4ry3-test-docker-image_tar_gz","label":":linux: aarch64-linux:test-docker-image.tar.gz"},{"command":"nix-store -r /nix/store/zjh5kllay6a2ws4w46267i97lrnyya9l-hello-2.12.1.drv","depends_on":[],"key":"zjh5kllay6a2ws4w46267i97lrnyya9l-hello-2_12_1","label":":linux: x86_64-linux:hello-2.12.1"},{"command":"nix-store -r /nix/store/6fh3frp0jqdnn4vi1ifdhxwzzx1igivi-synthetic-deps.drv","depends_on":["zjh5kllay6a2ws4w46267i97lrnyya9l-hello-2_12_1"],"key":"6fh3frp0jqdnn4vi1ifdhxwzzx1igivi-synthetic-deps","label":":linux: x86_64-linux:synthetic-deps"},{"command":"nix-store -r /nix/store/8kc33r36pvs3x6zm234drfzfpcm2pm4p-test-docker-image.tar.gz.drv","depends_on":["6fh3frp0jqdnn4vi1ifdhxwzzx1igivi-synthetic-deps","zjh5kllay6a2ws4w46267i97lrnyya9l-hello-2_12_1"],"key":"8kc33r36pvs3x6zm234drfzfpcm2pm4p-test-docker-image_tar_gz","label":":linux: x86_64-linux:test-docker-image.tar.gz"}]}